recipes/ner_pre_annotations_openai.ipynb

```diff
@@ -0,0 +1,1165 @@
+{
```

Comment:
<p>you can put the item title in bold: <strong>id:</strong> ...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/kili-technology/kili-python-sdk/pull/1075/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>